webpack.common.ts

```diff
@@ -30,6 +32,10 @@ module.exports = {
       src: path.resolve(__dirname, 'src'),
       assets: path.resolve(__dirname, 'assets'),
       react: path.resolve('./node_modules/react'),
+      vscode: path.resolve(
```

Comment:
what does this one do?

Reply:
Was a requirement of `monaco-languageclient` in order to set the correct vscode resolution. Their `0.19.0-next.1` version (which we are not using yet) explicitly does this for us. But earlier versions (like the 0.18.x that we are using) requires us to define this resolution in our own webpack config.

Testcase: the build had been failing without it as recently as `chore: final code cleanup` commit (when I confirmed all webpack changes were still required, by incrementing them on/off).

So it's required...
...until we switched to the `globalAPI` option today. Now everything appears to build.

Reply:
error if not present:
```
ERROR in ./node_modules/vscode-languageclient/lib/common/typeDefinition.js
Module not found: Error: Can't resolve 'vscode' in '/Users/denisewiedl/influxdata/ui/node_modules/vscode-languageclient/lib/common'
 @ ./node_modules/vscode-languageclient/lib/common/typeDefinition.js 8:17-34
 @ ./node_modules/monaco-languageclient/lib/monaco-language-client.js
 @ ./node_modules/monaco-languageclient/lib/index.js
 @ dll vendor
```